src/index.html

```diff
@@ -17,16 +17,24 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="search-wrapper">
+      <form class="search-bar search-bar--big" data-qa="big">
+        <input
+          class="search-bar__input--big"
+          type="text"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+        />
+      </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+      <form class="search-bar search-bar--small" data-qa="small">
+        <input
+          class="search-bar__input--small"
```

Comment:
The class names `search-bar__input--big` and `search-bar__input--small` are not very descriptive. It's better to use class names that describe the meaning of the content, not the styles. For example, you can use `search-bar__input-primary` and `search-bar__input-secondary`.